package.json

```diff
@@ -6,6 +6,7 @@
     "dev": "webpack-dev-server --config build-tools/config/webpack/webpack.config.js",
     "dev:code": "webpack --config build-tools/config/webpack/webpack.config.code.dev.js --watch",
     "build": "cross-env NODE_ENV=production node ./build-tools/script/build",
+    "postbuild": "cross-env node ./build-tools/script/diff2html",
```

Comment:
I think this script should not run automatically when running a build (build is run very often, but a delivery only every x days/weeks), so should be invoked manually.

Reply:
Ok. I think i will call it "build:diff" if you don't mind

Reply:
Sounds good!